src/main/resources/bidder-config/ttx.yaml

```diff
@@ -14,8 +14,8 @@ adapters:
       supported-vendors:
       vendor-id: 58
     usersync:
-      url:
-      redirect-url:
-      cookie-family-name: ttx
-      type: redirect
+      url: https://ic.tynt.com/r/d?m=xch&rt=html&gdpr={{gdpr}}&gdpr_consent={{gdpr_consent}}&ru=
+      redirect-url: /setuid?bidder=33across&uid=33XUSERID33X&id=zzz000000000002zzz
```

Comment:
Pls add `&gdpr={{gdpr}}&gdpr_consent={{gdpr_consent}}` GDPR macros.
Also, pls move query parameter `&id=zzz000000000002zzz` to `usersync.url` property as it is related to bidder exchange.